Dockerfile

```
@@ -8,8 +8,24 @@ RUN mkdir -p /devpi
 
 RUN devpi-init --serverdir /devpi
 
+COPY requirements.txt ./requirements.txt
+COPY start-devpi.sh ./start-devpi.sh
+RUN chmod +x start-devpi.sh
+RUN ./start-devpi.sh
+
+RUN chmod 777 -Rv /devpi/+files
+
+FROM python:3.10-alpine
+
+RUN pip install --quiet --upgrade devpi-server
```

nitpick, but seeing as each extra command creates a layer I usually try and keep amount of layers to a minimum:
```suggestion
RUN pip install --quiet --upgrade devpi-server && \
         devpi-server --version && \
         mkdir -p /devpi
```